core/settlement/position.go

```diff
@@ -61,48 +56,13 @@ func (t settlementTrade) MarketPrice() *num.Uint {
 	return t.marketPrice.Clone()
 }
 
-func newPos(evt events.MarketPosition) *pos {
+func newPos(marketPosition events.MarketPosition, price *num.Uint) *pos {
```

Comment:
Do we still rely on this type?

Reply:
in its simplified form yes. Because unfortunately we need to pass a MarketPosition to create a transfer.

Reply:
fair enough